telos-frontend/src/components/todo/DashboardTodo.jsx

```diff
@@ -38,62 +39,99 @@ const useStyles = makeStyles((theme) => ({
   datetextField: {
     marginLeft: theme.spacing(1),
     marginRight: theme.spacing(1),
-    width: 200,
+    width: '98%',
   },
 }));
 
-const listitems = [
-  {
-    name: 'OnGoing',
-    due: '18/03/2021',
-    onGoing: true,
-  },
-  {
-    name: 'OutDated',
-    due: '16/03/2021',
-    onGoing: false,
-  },
-];
-
 const outdated = {
   color: 'red',
 };
 
 const DashboardTodo = () => {
+  const listitems = [
+    {
+      name: 'OnGoing',
+      due: '2021-04-30',
+      onGoing: true,
+      completed: false,
+    },
+    {
+      name: 'OutDated',
+      due: '2021-01-30',
+      onGoing: false,
+      completed: true,
+    },
+    {
+      name: '一二三四五',
+      due: '2021-04-30',
+      onGoing: true,
+      completed: true,
+    },
+    {
+      name: '上山打老虎',
+      due: '2021-04-30',
+      onGoing: true,
+      completed: false,
+    },
+  ];
+
   const classes = useStyles();
-  const [checked, setChecked] = useState([0]);
   const [item, setItem] = useState('');
   const [dueDate, setDueDate] = useState('2021-01-01');
-  const [newItem, setNewItem] = useState(listitems);
+  const [itemList, setItemList] = useState(listitems);
+  const [newItem, setNewItem] = useState(itemList);
```

Comment:
what is the difference between itemList and listitems? Why do we need two different variables?